src/test/kotlin/com/atlassian/performance/tools/hardware/HardwareRecommendationIT.kt

```diff
@@ -32,7 +32,7 @@ class HardwareRecommendationIT {
         val aws = IntegrationTestRuntime.prepareAws()
         val engine = HardwareRecommendationEngine(
             product = PublicJiraSoftwareDistribution(jswVersion),
-            scale = ApplicationScales().extraLarge(jiraVersion = jswVersion, postgres = false),
+            scale = ApplicationScales().extraLarge(jiraVersion = jswVersion),
```

Comment:
Just raising a previous concern of mine.
Why do we constantly re-write 1 test?
Why aren't we sub-classing/extending this test and so keeping a history of tests run?

For example the _taskName = "QUICK-132-fix-v3"_ hasn't changed. Doesn't that cause potential clashes with existing cached data? 
e.g. either the test might overwrite existing data or not run everything because there is existing cached data? 

Or is my understanding wrong? (Always a high probability option :) )

Reply:
The easiest way to run a variation of the test is to modify it in-place. For ad-hoc "throwaway" tests, it's the path of least resistance.
AFAIK this ad-hoc test is not meant to be merged and got included in the PR by accident.
@swidnow2 how is it?

This opens discussion for:
* should we go for throwaway tests by default?
* should the reproducible tests encourage reproducible variations?
I think we'll find the time and place for this discussion during HWR retro.

Reply:
Let me make a note by saying:
- no, reproducibility we should have a collection
- yes, we should just have a separation by default though between "examples", "likely to be reused", and "would otherwise be thrown away, but might be valuable in the future"